content/blog/eks-oidc/index.md

```diff
@@ -0,0 +1,128 @@
+---
+date: "2020-05-08"
+title: "Access Control for Pods on AWS EKS"
+authors: ["mike-metral"]
+tags: ["EKS", "RBAC", "Kubernetes"]
+meta_desc: "AWS EKS clusters can use IAM roles and policies for Pods to assign fine-grained access control of AWS services."
+meta_image: cluster.svg
+---
+
+AWS [EKS][aws-eks] clusters can use [IAM][aws-iam] roles and policies for Pods
+to assign fine-grained access control of AWS services. The AWS IAM entities map into Kubernetes
+[RBAC][k8s-rbac] to configure the permissions of Pods that work with AWS
+services.
+
+Together, AWS IAM and Kubernetes RBAC enable least-privileged access for your
+apps, scoped to the appropriate policies and user requirements.
+
+[k8s-rbac]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+
+<!--more-->
+
+## Overview
+
+* [Pod Access Control](#pod-access-control)
+* [Create an OIDC provider](#create-an-oidc-provider)
+* [Create IAM for a S3 app](#create-iam-for-a-s3-app)
+* [Deploy a S3 app](#deploy-a-s3-app)
+* [Wrap-Up](#wrap-up)
+* [Next Steps](#next-steps)
+
+## Pod Access Control
+
+AWS EKS supports using IAM entities in a Pod [Service Account][k8s-sa] by
+leveraging an [OIDC provider][aws-oidc] connected to the Kubernetes cluster.
+
+Continuing with the example from the [AWS blog post][aws-pod-iam], when an [S3 app written in Go][s3-app] pushes
```

Comment:
Perhaps call out what this blog post is about, and how it relates to the post here?  Related - maybe highlight what this post is going to do as part of the overview above?  Perhaps using generally the same sentence as in the wrap-up?

> In <100 lines of code we've shown how EKS Pods can use AWS IAM to create fine-grained
permissions for apps that integrate with other AWS services.

Reply:
Done